### ISSUE TITLE
ansible-lint fails to start when git is not installed

### ISSUE THREAD

**Issue by fgierlinger** [ansible-community/ansible-lint#1464]
##### Summary

On a system with minimal package installed, `ansible-lint` fails to start due to a dependency to `git`. Even when only requesting the version of ansible-lint, an error of a missing dependency to git is shown.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
$ ansible --version
ansible 2.10.6
$ pip freeze | grep ansible
ansible-base==2.10.6
ansible-lint==5.0.3
$ ansible-lint --version
ansible-lint 5.0.3 using ansible 2.10.6
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

CentOS 8 as docker container

##### STEPS TO REPRODUCE

A container is used to simulate a system with minimal packages installed.
```console (paste below)
$ cat requirements.yml
---
roles:
  - src: geerlinguy.docker
$ cat playbook.yml
---
- hosts: all
  roles:
    - role: geerlingguy.docker
$ cat > Dockerfile <<EOF
FROM centos:8
RUN dnf install -y python3
RUN easy_install-3 pip
RUN pip3 install --upgrade pip
RUN pip3 install ansible-lint[core,yamllint]

COPY requirements.yml /requirements.yml
COPY playbook.yml /playbook.yml

CMD ['ansible-lint', '/playbook.yml']
EOF
$ podman build -q -t lint . 2>/dev/null
$ podman run --rm -ti lint
Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 8, in <module>
    sys.exit(_run_cli_entrypoint())
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 268, in _run_cli_entrypoint
    sys.exit(main(sys.argv))
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 169, in main
    initialize_options(argv[1:])
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 75, in initialize_options
    new_options = cli.get_config(arguments)
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/cli.py", line 377, in get_config
    options.config_file or f"{guess_project_dir()}/.ansiblelint"
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/file_utils.py", line 226, in guess_project_dir
    check=False,
  File "/usr/lib64/python3.6/subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib64/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'git': 'git'
```
The same error also shown when only requesting the version of `ansible-lint`
```console
$ podman run --rm -ti lint ansible-lint --version
Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 8, in <module>
    sys.exit(_run_cli_entrypoint())
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 268, in _run_cli_entrypoint
    sys.exit(main(sys.argv))
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 169, in main
    initialize_options(argv[1:])
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/__main__.py", line 75, in initialize_options
    new_options = cli.get_config(arguments)
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/cli.py", line 377, in get_config
    options.config_file or f"{guess_project_dir()}/.ansiblelint"
  File "/usr/local/lib/python3.6/site-packages/ansiblelint/file_utils.py", line 226, in guess_project_dir
    check=False,
  File "/usr/lib64/python3.6/subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib64/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'git': 'git'
```

##### Desired Behaviour

`ansible-lint` can be run without `git` being installed.

##### Actual Behaviour

`ansible-lint` should be able to be run without the dependency to git.

**Comment by aminvakil** [comment]
This has been fixed by https://github.com/ansible-community/ansible-lint/pull/1456, but it has not been included in `ansible-lint 5.0.3`.

**Comment by aminvakil** [comment]
@fgierlinger This should be fixed with latest release, https://github.com/ansible-community/ansible-lint/releases/tag/v5.0.4, check again.
